modules/ROOT/pages/deployment/services/s-list/proxy.adoc

```diff
@@ -23,6 +23,49 @@ The following request authentication schemes are implemented:
 -   Signed URL
 -   Public Share Token
 
+== Automatic User and Group Provisioning
+
+When using an external OpenID Connect IDP, the proxy can be configured to automatically provision users upon their first login.
+
+=== Prequisites
+
+A number of prerequisites must be met for automatic user provisioning to work:
+
+* Infinite Scale must be configured to use an external OpenID Connect IDP.
+* The `graph` service must be configured to allow updating users and groups (`GRAPH_LDAP_SERVER_WRITE_ENABLED`).
+* The IDP must return a unique value in the user's claims (as part of the userinfo response and/or the access tokens) that can be used to identify the user. This claim needs to be stable and cannot be changed for the whole lifetime of the user. That means, if a claim like `email` or `preferred_username` is used, you must ensure that the user's email address or username never changes.
```

Comment:
> you must ensure that the user's email address or username never changes.

Later you state:

> If the user does already exist, the proxy will check if the user's email or displayname has changed and updates those accordingly via `graph` service.

So what is it now? Do I need to take care the email never changes or will it be updated automatically?

Reply:
The text is taken from https://github.com/owncloud/ocis/pull/9458 which was created by @rhafer ... 
If a fix is needed, we also need to do that in the ocis repo proxy service readme...

Reply:
As I understand it, the "claim" can be set to one of various data items. The selected data item then has to be a unique and unchanging "key" for the user. So, if `email` is selected, then the email address is the unique thing that identifies each user, and so changing the email address in the IDP is going to "lose" the user in ocis. But if some other data item is used as the "claim" unique identifier for the user, the the email address is just a piece of data about the user, and can change.

Reply:
The discussion shows that the text is not clear and must be fixed. I have filed an issue in the ocis repo.